packages/features/liquidity/src/liquidity-variable/hooks/use-apy-chart.tsx

```diff
@@ -79,6 +81,29 @@ export const useApyChart = (token?: TokenDefinition, defaultDataLimit = 50) => {
     });
   }
 
+  if (selectedNetwork === Network.arbitrum) {
```

Comment:
This code was changed on alpha/arbitrum, you should rebase that because this logic is incorrect.